tests/test_pytest_qgis.py

```diff
@@ -119,3 +119,9 @@ def test_iface_toolbar_qtoolbar(qgis_iface):
     qgis_iface.addToolBar(toolbar)
     assert toolbar.windowTitle() == name
     assert qgis_iface._toolbars == {name: toolbar}
+
+
+def test_canvas_should_be_released(qgis_canvas, layer_polygon, layer_points):
```

Comment:
Could you please add a clarifying comment for this test that how it is testing since it does not have any asserts.

Reply:
Sure, I'll clarify this.